{"version":3,"file":"index.cjs91.js","sources":["../node_modules/axios/lib/cancel/CanceledError.js"],"sourcesContent":["'use strict';\n\nimport AxiosError from '../core/AxiosError.js';\nimport utils from '../utils.js';\n\n/**\n * A `CanceledError` is an object that is thrown when an operation is canceled.\n *\n * @param {string=} message The message.\n * @param {Object=} config The config.\n * @param {Object=} request The request.\n *\n * @returns {CanceledError} The created error.\n */\nfunction CanceledError(message, config, request) {\n  // eslint-disable-next-line no-eq-null,eqeqeq\n  AxiosError.call(this, message == null ? 'canceled' : message, AxiosError.ERR_CANCELED, config, request);\n  this.name = 'CanceledError';\n}\n\nutils.inherits(CanceledError, AxiosError, {\n  __CANCEL__: true\n});\n\nexport default CanceledError;\n"],"names":["CanceledError","message","config","request","AxiosError","utils"],"mappings":"8KAcA,SAASA,EAAcC,EAASC,EAAQC,EAAS,CAE/CC,EAAAA,QAAW,KAAK,KAAMH,GAAkB,WAAsBG,EAAU,QAAC,aAAcF,EAAQC,CAAO,EACtG,KAAK,KAAO,eACd,CAEAE,EAAAA,QAAM,SAASL,EAAeI,UAAY,CACxC,WAAY,EACd,CAAC","x_google_ignoreList":[0]}